swagger.yaml

```diff
@@ -4856,6 +4929,14 @@ definitions:
         type: string
         description: Json formatted string containing ip collisions detected in the cluster.
         x-go-custom-tag: gorm:"type:text"
+      ignored_host_validations:
+        type: string
+        description: Json formatted string containing a list of host validations to be ignored. May also contain a list with a single string "all" to ignore all host validations. Some validations cannot be ignored.
```

Comment:
```suggestion
        description: JSON-formatted list of host validation IDs that will be ignored.  It may also contain a list with a single string "all" to ignore all host validations.  Some validations cannot be ignored.
```